keyboards/wsk/jerkin/config.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```

Comment:
`echo -n "jerkin" | shasum | cut -c1-4`
```suggestion
#define PRODUCT_ID      0x79AE
```